src/main/java/uk/gov/hmcts/reform/bulkscanprocessor/entity/Status.java

```diff
@@ -38,7 +38,7 @@
     }
 
     public boolean isProcessed() {
-        return this == PROCESSED || this == NOTIFICATION_SENT;
+        return this == PROCESSED || this == NOTIFICATION_SENT || this == COMPLETED;
```

Comment:
can you explain?
what does 'processed' actually mean and when is it used?

Reply:
adding javadoc to this method would be great, I don't think it's clear with those 3 different statuses, out of which one is called 'processed'

Reply:
also, probably good idea to rename this method

Reply:
I'll add a comment. The name of me method may be right - all those statuses represent a processed envelope, from processor's perspective. And `PROCESSED` itself is probably going away, as it duplicates another status - `UPLOADED`.